contrib/libpng-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# option(ENABLE_LIBPNG "Enable libpng" ${ENABLE_LIBRARIES})
```

Comment:
Not making libpng dependent on ENABLE_LIBRARIES will compile it unconditionally in all builds. This isn't good, FastTest does a minimal build with as few as possible library dependencies. Let's restore l. 1-6.